pyvista/plotting/_property.py

```diff
@@ -257,6 +264,8 @@ def __init__(
             edge_opacity = self._theme.edge_opacity
         self.edge_opacity = edge_opacity
 
+        self.shading = shading
```

Comment:
Should probably add `index_of_refraction` and `anisotropy_rotation` here as well